src/components/fixes/PatchedExternalLink.tsx

```diff
@@ -0,0 +1,15 @@
+import { ForwardRefRenderFunction, forwardRef } from 'react'
+
+// https://headlessui.com/react/menu#integrating-with-next-js
+const PatchedExternalLink: ForwardRefRenderFunction<
```

Comment:
**question** could we make this change to the existing ExternalLink component instead?

Reply:
This is a very specific thing where it is forwarding the ref I believe. I don't know what the wider consequences might be so i'd prefer to just do it here for now. THis is fixed in Next13 so when we eventually upgrade this will go away